website/src/components/Visualisations/Roles/Roles.tsx

```diff
@@ -45,10 +45,12 @@ const CustomTooltip = ({ active, label, data, colors }: CustomTooltipProps) => {
           <p
             className={styles.content}
             key={team}
-            style={{ color: colors.get(team) }}
+            style={{ color: teamColors.get(team) }}
```

Comment:
is this ok with this returning undefined btw?

Reply:
yep, color prop is ok with undefined.